docs/skillmap/story/story1.md

```diff
@@ -1,6 +1,19 @@
 # Greeting Card
 
-### @autoexpandOff true
+
+```assetjson
```

Comment:
are you seeing some issue with the tutorials that requires the assetjson to be at the top of the file?

Reply:
It's an issue with the tutorial-tool.  Sometimes it won't read it at the bottom.

Reply:
hmm the tutorial tool is just loading the arcade website in an iframe, so if there's an issue there it will probably also be present in the livesite. if you see it again could you save the markdown that's causing the issue and file a bug/ping me? i was looking into this issue with galga but it stopped reproing, haha